### DISCUSSION TITLE
Error cuando ejecuto el comando mvn

### DISCUSSION THREAD

**Discussion by nicolasgalfione** [dds-jv/foro#352] [Consultas de herramientas] [answered]
![image](https://user-images.githubusercontent.com/82039967/164891797-2a892997-0d0b-4e97-82ed-ffc1022f4b21.png)

hola! cuando intento ejecutar el comando mvn test o el verify me tira este warning y me dice que no detecta ningun test, todo esto seguido de muchos errores pero lo principal es eso de la foto

no se si sera problema de maven o su version asi que aca dejo por las dudas la version que me indica que tengo instalada
![image](https://user-images.githubusercontent.com/82039967/164891948-53f4c6f8-3039-47d9-8efd-949947b0ff79.png)

**Comment by gastonprieto** [comment, marked as answer]
Buenas @nicolasgalfione !

que versión de java estas usando? la 8? porque el proyecto esta configurado para ejecutar la 8

**Reply by nicolasgalfione** [reply]
tenia mal la version y la cambie a la 8 pero me sigue tirando esos errores

**Reply by gastonprieto** [reply]
si haces el `mvn --version` que verrsión te da ? podras pegar el error nuevamente para ver si cambio algo ? fijate  el error que dice que jdk usa, tal vez sigue usando la 18

**Reply by nicolasgalfione** [reply]
ahi lo solucione profe, gracias por la ayuda!

**Reply by gastonprieto** [reply]
genial, que era ?

**Reply by nicolasgalfione** [reply]
Tuve que desistalar la versión de java 18 porque me la seguía tomando y después ahí si con la única que tenia la 8 ya me fue bien
